js/tsconfig.json

```diff
@@ -7,7 +7,8 @@
   "compilerOptions": {
     "target": "ESNEXT",
     "module": "commonjs",
-    "noEmit": true
+    "noEmit": true,
+    "esModuleInterop": true
```

Comment:
Just curious, is this necessary for this PR, or is it just something we want to do anyway (I see it's recommended [here](https://www.typescriptlang.org/tsconfig#esModuleInterop))

Reply:
Ahh, so this is the tsconfig file that editors like VSCode will pick up. We don't use it to compile things but just for type checking. esModuleInterop is needed for kleur in the perf tests. 

<img width="861" alt="Screen Shot 2021-05-22 at 11 00 08" src="https://user-images.githubusercontent.com/589034/119236548-e0b5ef80-baec-11eb-92e5-5920a22ecfb2.png">

Reply:
Got it, thanks

Reply:
Having a single tsconfig for the editor that type checks everything is super convenient as we can see all references across the whole codebase including from tests and now benchmarks.